### DISCUSSION TITLE
Gestion des non-conformités

### DISCUSSION THREAD

**Discussion by dbarzin** [dbarzin/deming#67] [Ideas]
Nous envisageons d'ajouter une fonctionnalité cruciale à notre outil en intégrant la gestion des non-conformités. Nous souhaitons ouvrir une discussion pour recueillir vos retours et suggestions sur les champs que vous considérez comme essentiels pour cette nouvelle fonctionnalité.

Actuellement, notre objectif est de permettre la saisie des éléments suivants :

- Nom
- Type (non-conformité mineure/majeure, observation, opportunité d'amélioration, point sensible)
- Description
- Responsables (liste d'utilisateurs de l'application)
- Date de création
- Analyse de la cause racine
- Action corrective
- Correction de la non-conformité
- Date d'échéance
- Date de clôture
- Justification de la clôture

L'intégration de cette fonctionnalité dans notre outil permettra :

- D'ajouter une date de réalisation, intégrée dans le plan d'action à moyen ou court terme.
- De recevoir des notifications par mail pour les échéances à venir.
- D'être inclus dans les rapports et discuté/communiqué lors des revues de direction.
- D'être pris en compte pour les preuves d'amélioration continue.

Nous aimerions avoir vos réflexions sur les champs proposés et savoir si vous suggérez l'ajout de tout autre champ pertinent pour votre utilisation quotidienne. Votre retour est essentiel pour nous aider à concevoir une fonctionnalité qui répond au mieux à vos besoins.

N'hésitez pas à partager vos idées, expériences et tout autre élément que vous estimez important pour cette gestion des non-conformités. Vos retours contribueront à renforcer l'efficacité de Deming.

**Comment by thibaudba** [comment]
Très bonne idée de fonctionnalité ! D'un point de vue opérationnel, c'est une fonctionnalité qui sera particulièrement utile, notamment pour faire un suivi de ces non-conformités sur des périmètres précis. 

Quelques idées : 
- Un lien direct vers le(s) critère(s) / le(s) référentiel(s) associés
- Rendre le moins de champs possibles obligatoires (on souhaite utiliser Deming sur plusieurs périmètres qui ne sont pas tous 27001)
- Une vision qui serait intéressante serait d'avoir une liste des non-conformité avec une possibilité de filtrer sur les périmètres et sur le type
- Eventuellement de pouvoir préciser la source de l'identification de la non-conformité (outil interne, audit, contrôle, remontée des équipes, etc.)

Un autre type de non-conformité qui aurait été utile pour nous dans certains cas : la dérogation. Nous importons nos référentiels internes dans l'outil, et dans certains cas nous estimons que la réponse technique apportée par les équipes n'est pas conforme avec ce que dit notre référentiel mais qu'après analyse, on considère qu'il n'y a pas d'impact / que le niveau de sécurité apporté est le même que celui de la mesure que nous préconisons. Le suivi de ces dérogations est une bonne source pour faire évoluer nos référentiel. Mais peut-être serait-ce un doublon par rapport à certains types que vous imaginez ou peut-être sort-on un peu trop du cadre d'usage prévu pour Deming.

**Reply by dbarzin** [reply]
Très intéressant, merci.

**Comment by dbarzin** [comment]
:tada: Cette évolution est disponible :tada:
